src/Services/ArrayManager.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Helpers\Services;
+
+class ArrayManager
+{
+    /**
+     * Returns TRUE if var is an array, FALSE otherwise.
+     * @param array $value
+     * @return bool
+     */
+    public function isArray(array $value): bool
+    {
+        if (count($value) === 0) {
```

Comment:
If we pass empty array, It should also return as Array, then what is the purpose of adding **count**. We should consider empty arrays also.